service-onboarding-build-pack/Jenkinsfile

```diff
@@ -13,23 +13,7 @@ import groovy.transform.Field
 -git push template to git
 */
 
-//definitions
-def Event_Name = [
```

Comment:
Why remove this enumeration? you can always expand this to add the mission event names